app/org/sagebionetworks/bridge/services/TaskService.java

```diff
@@ -170,10 +168,9 @@ public void deleteTasks(String healthCode) {
     }
     
     private Activity createResponseActivityIfNeeded(StudyIdentifier studyIdentifier, String healthCode, Activity activity) {
-        if ((activity.getActivityType() != ActivityType.SURVEY)) {
+        if (activity.getActivityType() != ActivityType.SURVEY || activity.getSurveyResponse() != null) {
```

Comment:
Doe this mean that an activity may have a survey response reference but not a survey activity?  I have to read the documentation which says "any activity that has a SurveyReference will also contain a SurveyResponseReference object".